From 7eea072083f0a24589f65d8a480118738b1d1156 Mon Sep 17 00:00:00 2001
From: Tang Yuantian <andy.tang@nxp.com>
Date: Thu, 6 Sep 2018 09:42:01 -0400
Subject: [PATCH 281/706] arm64: dts: fsl: add GPU node in LS1028A

add device tree nodes for GPU in LS1028A to enable the GPU
feature in LS1028A SOC based boards like RDB and QDS boards.

Signed-off-by: Tang Yuantian <andy.tang@nxp.com>
(cherry picked from commit cf5d3cb0741197b9e99fde2af97c4632918c46d1)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
index d1f9815c204c..7c68c8ed617b 100755
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -102,6 +102,16 @@
 		device_type = "memory";
 	};
 
+	gpu@f0c0000 {
+		compatible = "fsl,ls1028a-gpu";
+		reg = <0x0 0x0f0c0000 0x0 0x10000>,
+			<0x0 0x80000000 0x0 0x80000000>,
+			<0x0 0x0 0x0 0x3000000>;
+		reg-names = "base", "phys_baseaddr",
+			"contiguous_mem";
+		interrupts = <0 220 0x4>;
+	};
+
 	sysclk: sysclk {
 		compatible = "fixed-clock";
 		#clock-cells = <0>;
-- 
2.17.1

